cli/tests/integration_tests.rs

```diff
@@ -5540,25 +5540,6 @@ console.log("finish");
     assert!(stderr.contains("BadResource"));
   }
 
-  #[cfg(not(windows))]
-  #[test]
-  fn should_not_panic_on_not_found_cwd() {
-    let output = util::deno_cmd()
-      .current_dir(util::root_path())
-      .arg("run")
-      .arg("--allow-write")
-      .arg("--allow-read")
-      .arg("cli/tests/dont_panic_not_found_cwd.ts")
-      .stderr(std::process::Stdio::piped())
-      .spawn()
-      .unwrap()
-      .wait_with_output()
-      .unwrap();
-    assert!(!output.status.success());
-    let stderr = std::str::from_utf8(&output.stderr).unwrap().trim();
-    assert!(stderr.contains("Failed to get current working directory"));
-  }
-
```

Comment:
Intentional?

Reply:
It is, this test was actually testing `Worker::execute()` that was reading CWD without good reason.